packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3392,10 +3392,15 @@ export class ContainerRuntime
 		// The summary number for this summary. This will be updated during the summary process, so get it now and
 		// use it for all events logged during this summary.
 		const summaryNumber = this.nextSummaryNumber;
+		let summaryRefSeqNum: number | undefined;
 		const summaryNumberLogger = createChildLogger({
 			logger: summaryLogger,
 			properties: {
-				all: { summaryNumber },
+				all: {
+					summaryNumber,
+					summaryReferenceSequenceNumber: () => summaryRefSeqNum,
+					initialSequenceNumber: this.deltaManager.initialSequenceNumber,
```

Comment:
FYI - Container already records this (see dmInitialSeqNumber), though only on errors. Also Container:Load_end will also have sequenceNumber property that is a baseline of snapshot (same thing).

Should this be recorded instead as part of GCFeatureMatrix (maybe renamed to GCInitialState) or better - as part of ContainerLoadStats? It's not GC specific, and useful in general, so I'd keep it there?